docs/how-to-use-minio-as-laravel-file-storage.md

```diff
@@ -0,0 +1,102 @@
+# How to use Minio Server as [Laravel](https://laravel.com) Custom File Storage 
+
```

Comment:
Is it required to have laravel configured to use this document? i am not able to verify this BTW.

Reply:
Laravel has its own document for implementing & configuring it to use custom storage driver [for 5.5](https://laravel.com/docs/5.5/filesystem#custom-filesystems) & [for 5.4](https://laravel.com/docs/5.4/filesystem#custom-filesystems) & [for 5.3](https://laravel.com/docs/5.3/filesystem#custom-filesystems). all of this are the same & i wrote this recipe similar to this document.

Reply:
Do you have a skeletal laravel project available? which i can use to test?

Reply:
Sorry for my late.
I create a sample and pushed to  [laravel-minio-sample](https://github.com/m2sh/laravel-minio-sample) repo.
It has some [tests](https://github.com/m2sh/laravel-minio-sample/blob/master/tests/Unit/MinioStorageTest.php) that shows how to use minio with laravel and you can see all tests has been [passed successfully](https://travis-ci.org/m2sh/laravel-minio-sample).

Reply:
This looks good @m2sh - can you reference this repo in the document? it would make it easier for users to see a finished project structure to get started.

Reply:
Done.
I added the reference of repo to document.